doc/event-alarm-framework/event-alarm-framework.md

```diff
@@ -322,72 +300,22 @@ Developers of new events or alarms need to update this file by declaring name an
     ]
 }
 ```
-The format of event notify API is:
-
-definition:
-```
-    LOG_EVENT(name, source, action, MSG, ...)
-```
-- name is name of the event
-- source is the object that is generating this event
-- action is either NOTIFY, RAISE_ALARM, CLEAR_ALARM, ACK_ALARM or UNACK_ALARM
-- MSG can be json string. If json string, it is rendered as is in the syslog.
-
-
-Usage:
-For one-shot events:
-```
-    LOG_EVENT(CUSTOM_EVPROFILE_CHANGE, profile_name.c_str(), NOTIFY, "New event profile is %s", profile_name.c_str());
-```
 
-For alarms:
-```
-    if (temperature >= THRESHOLD) {
-        LOG_EVENT(TEMPERATURE_EXCEEDED, sensor_name_p, RAISE_ALARM, "Temperature for sensor %s is %d degrees", sensor_name_p, current_temp);
-    } else {
-        LOG_EVENT(TEMPERATURE_EXCEEDED, sensor_name_p, CLEAR_ALARM, "Temperature for the sensor %s is %d degrees ", sensor_name_p, current_temp);
-    }
-```
 #### 3.1.1.2 Development Process
```

Comment:
It seems like this is introducing differences in the way that an application/component needs to implement their events and alarms. 
If this is the case, we should clearly outline what these changes are, ideally prepare an example, and raise this in the Community during one of the Tuesday calls. 
If you agree, we can work with @zhangyanzhao to schedule something.

Reply:
Resolved.